drivers/veteran_confirmation/app/models/veteran_confirmation/va_check_history.rb

```diff
@@ -5,11 +5,34 @@
 ###
 
 module VeteranConfirmation
-  class Checker
+  class VaCheckHistory < GrdaWarehouseBase
+    self.table_name = :va_check_histories
+
+    # Response values
     CONFIRMED = 'confirmed'.freeze
     NOT_CONFIRMED = 'not confirmed'.freeze
 
-    def check(clients)
+    belongs_to :client, class_name: 'GrdaWarehouse::Hud::Client'
+
+    scope :recent, -> { order(check_date: :desc).limit(1) }
+
+    def occured_within(days)
+      check_date.present? && check_date <= Date.current - days
+    end
+
+    def check
+      query_results = self.class.check(::GrdaWarehouse::Hud::Client.where(id: client_id))
+      update!(
+        response: query_results[client_id],
+        check_date: Date.current,
+      )
```

Comment:
I'm guessing we want some error correction here or in `self.check` that bails (and logs) if it fails to successfully call the API.